exchange.c

```diff
@@ -47,8 +50,8 @@ int domain_exchange(int (*layoutfunc)(int p), int failfast) {
     /* flag the particles that need to be exported */
     int * toGo_arr[NSP];
     int * toGet_arr[NSP];
-    toGo_arr[0] = (int *) mymalloc("toGo", (NSP * sizeof(int) * NTask));
-    toGet_arr[0] = (int *) mymalloc("toGet", (NSP * sizeof(int) * NTask));
+    toGo_arr[0] = (int *) mymalloc2("toGo", (NSP * sizeof(int) * NTask));
```

Comment:
keep in mind NTask can fetch to millions in the near future.

Reply:
Sure, but why does this matter? The memory is the same no matter whether it is top or bottom